DAQ_System/Source/main.cpp

```diff
@@ -46,122 +48,57 @@ int main() {
+        FileWrite(data_file, buffer);
+        RPM += 100;
+        tire_temp += 10;
+        air_temp++;
+        battery -= 10;
+        timer = clock() - timer;
     }
 
-    printf("\rWriting numbers (%d/%d)... OK\n", 10, 10);
-
-    // create a separate function for this as well maybe?
-    printf("Seeking file... ");
-    fflush(stdout);
-    status = fseek(numbers_file, 0, SEEK_SET);
-    printf("%s\n", (status < 0 ? "Fail :(" : "OK"));
-    if (status < 0) {
-        error("error: %s (%d)\n", strerror(errno), -errno);
-    }
-
-    long pos = 0;
-
-    // Go through and increment the numbers
-    for (int i = 0; i < 10; i++) {
-        printf("\rIncrementing numbers (%d/%d)... ", i, 10);
-        fflush(stdout);
-
-        // Get current stream position
-        pos = ftell(numbers_file);
-
-        // Parse out the number and increment
-        int32_t number;
-        fscanf(numbers_file, "%ld", &number);
-        number += 1;
-
-        // Seek to beginning of number
-        fseek(numbers_file, pos, SEEK_SET);
-
-        // Store number
-        sprintf(buffer, "    %d\n", number);
-        FileWrite(numbers_file, buffer);
-
-        // Flush between write and read on same file
-        fflush(numbers_file);
-    }
-
-    printf("\rIncrementing numbers (%d/%d)... OK\n", 10, 10);
-
     // Close the file which also flushes any cached writes
-    printf("Closing \"/fs/numbers.txt\"... ");
+    printf("Closing \"/fs/data.csv\"... ");
     fflush(stdout);
-    FileClose(numbers_file);
-
-
-    // Display the root directory
-    printf("Opening the root directory... ");
-    fflush(stdout);
-
-    DIR* root_dir = opendir("/fs/");
-    printf("%s\n", (!root_dir ? "Fail :(" : "OK"));
-    if (!root_dir) {
-        error("error: %s (%d)\n", strerror(errno), -errno);
-    }
-
-    printf("root directory:\n");
-    while (true) {
-        struct dirent* entry = readdir(root_dir);
-        if (!entry) {
-            break;
-        }
-
-        printf("    %s\n", entry->d_name);
-    }
-
-    printf("Closing the root directory... ");
-    fflush(stdout);
-    status = closedir(root_dir);
-    printf("%s\n", (status < 0 ? "Fail :(" : "OK"));
-    if (status < 0) {
-        error("error: %s (%d)\n", strerror(errno), -errno);
-    }
-
-    // Display the numbers file
-    printf("Opening \"/fs/numbers.txt\"... ");
-    fflush(stdout);
-    numbers_file = FileOpen("/fs/numbers.txt");
-
-    printf("numbers:\n");
-    while (!feof(numbers_file)) {
-        int num = fgetc(numbers_file);
-        printf("%c", num);
-    }
-
-    printf("\rClosing \"/fs/numbers.txt\"... ");
-    fflush(stdout);
-    FileClose(numbers_file);
+    FileClose(data_file);
 
     // Tidy up
     printf("Unmounting... ");
     fflush(stdout);
     Unmount(&file_system);
-    
+
+    // not really sure if the following block device stuff is important to the function of the program or not so I left it alone, but I don't think it needs to be here
```

Comment:
Just documented the minimal implementation after recent progress: https://github.com/DallasFormulaRacing/DataAcquisition2.0/wiki/Minimal-Implementations

You can remove this code section off your work